ReactWindows/ReactNative.Net46/ReactPage.cs

```diff
@@ -241,4 +241,4 @@ private void ApplyArguments(string[] arguments)
             }
         }
     }
-}
+}
```

Comment:
I think in general we've been leaving newlines at the end of files, but I'm sure that's not actually consistent.